dashboard/app/controllers/api/v1/users_controller.rb

```diff
@@ -20,7 +20,8 @@ def current
         id: current_user.id,
         username: current_user.username,
         user_type: current_user.user_type,
-        is_signed_in: true
+        is_signed_in: true,
+        short_name: current_user.short_name
```

Comment:
Is there a corresponding test that we should update?

Reply:
I updated a test in users_controller_test